contracts/eigenlayer/src/handlers/MirrorServiceHandler.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IWavsServiceHandler} from "../../../interfaces/IWavsServiceHandler.sol";
+import {IWavsServiceManager} from "../../../interfaces/IWavsServiceManager.sol";
+import { MirrorStakeRegistry } from "../MirrorStakeRegistry.sol";
+
+
+interface IMirrorUpdateTypes {
+    error InvalidTriggerId(uint64 expectedTriggerId);
+
+    /// @notice DataWithId is a struct containing a trigger ID and updated operator info
+    struct UpdateWithId {
```

Comment:
Is this all the items to update that affect `isValidSignature`?

I think there is a theshold weight as well. Can you add this?